css/font-awesome.css

```diff
@@ -61,7 +61,7 @@
 }
 .fa-border {
   padding: .2em .25em .15em;
-  border: solid 0.08em #eeeeee;
+  border: solid 0.08em #eee;
```

Comment:
Sure, I included it in a separate commit (https://github.com/FortAwesome/Font-Awesome/pull/9586/commits/1dbb74d666d1906d77abe86b3c6ce9e9dd5071cf) - I'll remove that and open a new PR to abbreviate those values.